data/tlg0061/tlg005/tlg0061.tlg005.1st1K-grc1.xml

```diff
@@ -0,0 +1,354 @@
+                <author>Lucian</author>
+                <editor>Karl Jacobitz</editor>
+                <imprint>
+                    <publisher>Teubner</publisher>
+                    <pubPlace>Leipzig</pubPlace>
+                    <date>1913</date>
+                </imprint>
+                <biblScope unit="volume">3</biblScope>
+            </monogr>
+            <ref target=" https://archive.org/details/operaexrecogniti03luciuoft/page/454/mode/2up">Internet Archive</ref>
+        </biblStruct>
+    </sourceDesc>
+</fileDesc>
+     
+     <encodingDesc>
+         <editorialDecl>
+             <correction cert="medium">
+                 <p>Greek produced via Lace v. 0.6.22 copyright 2013-2021, Bruce Robertson, Dept. of Classics, Mount Allison University. Developed through the support of the <ref target="https://nbif.ca/en">NBIF</ref>.</p>
+                 <p>Needs further proofreading refinement. Oddities in OCR capture include ζ/ξ confusion, χ/κ confusion, capitalization errors, missed hyphenization at line breaks and page breaks, word transposition after em dashes, punctuation confusion where comma is read as period and Greek semi-colon is misread.</p>
+             </correction>
+         </editorialDecl>
+         
+         <p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p>
+         
+         <refsDecl n="CTS">
+             <cRefPattern n="line" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div//tei:l[@n='$1'])">
+                 <p>This pointer pattern extracts line</p>
+             </cRefPattern>
+         </refsDecl>
+     </encodingDesc>
+     
+     <profileDesc>
+         <langUsage>
+             <language ident="grc">Greek</language>
+         </langUsage>
+     </profileDesc>
+     
+     <revisionDesc>
+         <change who="Lisa Cerrato" when="2023-07-17">File creation, initial OCR editing, EpiDoc and CTS conversion.</change>  </revisionDesc>
+ </teiHeader>
+   
+<text>
+<body>
+<div type="edition" n="urn:cts:greekLit:tlg0061.tlg005.1st1K-grc1" xml:lang="grc">
+<pb facs="operaexrecogniti03luciuoft_0491"/>
+<l n="0" rend="align(indent)">Ὠκύπους Ποδαλειρίου καὶ Ἀστασίας υἱὸς ἐγένετο, κάλλει καὶ δυνάμει διαφέρων, γυμνασίων τε καὶ κυνηγεσίων μὴ ἀμελῶν. πολλάκις δὲ θεωρῶν τοὺς ἐχομένους ὑπὸ τῆς ἀτέγκτου Ποδάγρας κατεγέλα φάσκων μηδὲν ὅλως εἶναι τὸ πάθος. ἡ θεὸς τοίνυν ἀγανακτεῖ καὶ διὰ ποδῶν εἰστρέχει. τοῦ δ’ εὐτόνως φέροντος καὶ ἀρνουμένου, ὕπτιον ὅλως τίθησιν ἡ θεός. ἡ μὲν σκηνὴ τοῦ δράματος ὑπόκειται ἐν Θήβαις· ὁ δὲ χορὸς συνέστηκεν ἐξ ἐπιχωρίων ποδαγρῶν. συνελεγχόντων τὸν Ὠκύπουν. τὸ δὲ δρᾶμα τῶν πάνυ ἀστείων. τὰ τοῦ δράματος πρόσωπα Ποδάγρα. Ὠκύπους, Τροφεύς, Ἰατρός, Πόνος, Ἄγγελος. προλογίζει δὲ ἡ Ποδάγρα.</l>
+
+<sp><speaker>Ποδάγρα</speaker>
+    <l n="1" xml:base="urn:cts:greekLit:tlg0061.tlg005.1st1K-grc1"> <milestone unit="Reitz_page" n="665"/> Δεινὴ μὲν ἐν βροτοῖσι καὶ δυσώνυμος</l>
```

Comment:
hi @lcerrato this more a question about xml:base than a comment, why does the xml:base repeat the exact same base URN through this file rather than changing, the way it does in other XML files?